packages/vue/src/stories/releases/v0.10.0/migration.stories.mdx

```diff
@@ -394,4 +389,33 @@ It was replaced by grid styling set directly in the inline CSS like this:
       <slot />
     </div>
   </footer>
-```
\ No newline at end of file
+```
+
+## Atoms refactored
+
+As the better web performance is one of our main goals, we refactored components without additional logic into functional components.
+This is the list of changed components:
+
+- [SfArrow](?path=/story/components-atoms-arrow--common)
+- [SfBadge](?path=/story/components-atoms-badge--primary)
+- [SfChevron](?path=/story/components-atoms-chevron--common)
+- [SfCircleIcon](?path=/story/components-atoms-circleicon--primary)
+- [SfLoader](?path=/story/components-atoms-loader--loading)
+- [SfOverlay](?path=/story/components-atoms-overlay--common)
+- [SfPrice](?path=/story/components-atoms-price--common)
+- [SfProperty](?path=/story/components-atoms-property--common)
+- [SfTextarea](?path=/story/components-atoms-textarea--common)
+
+So the example component is now looking like this:
+
+```
+<template functional>
+  <div
+    :class="[data.class, data.staticClass, 'sf-badge']"
+    :style="[data.style, data.staticStyle]"
```

Comment:
remove `data.class` and `data.staticClass` from this example.

Reply:
removed